docs/blog/announcing-vite3.md

```diff
@@ -0,0 +1,228 @@
+---
+sidebar: false
+---
+
+# Vite 3.0 is out!
+
+In February last year, [Evan You](https://twitter.com/youyuxi) released Vite v2. Since then, its adoption has grown non-stop, reaching more than 1 million npm downloads per week. A sprawling ecosystem rapidly formed after the release. Vite is powering a renewed innovation race in Web frameworks. [Nuxt v3](https://v3.nuxtjs.org/) will use Vite by default. [SvelteKit](https://kit.svelte.dev/), [Astro](https://astro.build/), [Hydrogen](https://hydrogen.shopify.dev/), and [SolidStart](https://docs.solidjs.com/start) are all built with Vite. [Laravel has now decided to use Vite by default](https://laravel.com/docs/9.x/vite). [Vite Ruby](https://vite-ruby.netlify.app/) shows how Vite can improve Rails DX. [Vitest](https://vitest.dev) is making strides as a Vite-native alternative to Jest. Vite is behind [Cypress](https://docs.cypress.io/guides/component-testing/writing-your-first-component-test) and [Playwright](https://playwright.dev/docs/test-components)'s new Component Testing features, Storybook has [Vite as an official builder](https://github.com/storybookjs/builder-vite). And [the list goes on](https://patak.dev/vite/ecosystem.html). Maintainers from most of these projects got involved in improving the Vite core itself, working closely with the Vite [team](https://vitejs.dev/team) and other contributors.
+
+Today, 16 months from the v2 launch we are happy to announce the release of Vite 3. We decided to release a new Vite major at least every year to align with [Node's EOL](https://nodejs.org/en/about/releases/), and take the opportunity to review Vite's API regularly with a short migration path for projects in the ecosystem.
```

Comment:
Maybe we should refer "Node" as "Node.js", and "Vite v3" as "Vite 3", for the entire article. Me and sapphi briefly discussed this at https://github.com/vitejs/vite/pull/9030#issuecomment-1181267590, though it's a minor nitpick.

Reply:
Same for Nuxt v3 -> Nuxt? Ya, I'll change it

Reply:
done